src/components/Proposals/Proposals.js

```diff
@@ -1,15 +1,36 @@
 import React from 'react'
-import { Link } from 'react-router-dom'
+import { CardLayout, GU, useLayout } from '@aragon/ui'
+import { createAppHook, useApp } from '@aragon/connect-react'
+import connectVoting from '@aragon/connect-voting-disputable'
+import ProposalCard from './ProposalCard'
+
+const useDisputableVoting = createAppHook(connectVoting)
 
 function Proposals() {
+  const [disputableVoting] = useApp('disputable-voting')
+  const [votes] = useDisputableVoting(disputableVoting, (app) => app.votes())
+  const { layoutName } = useLayout()
+  const compactMode = layoutName === 'small'
+  const rowHeight = compactMode ? null : 294
+
   return (
     <>
-      Proposals view
-      <div>
-        <Link to="/proposals/1">Proposal 1</Link>
-        <Link to="/proposals/2">Proposal 2</Link>
-        <Link to="/proposals/3">Proposal 3</Link>
-      </div>
+      <section
+        css={`
+          width: 80vw;
+          margin: auto;
+        `}
+      >
```

Comment:
Just a heads up that we will probably be limiting our layout in the `MainView` so we can remove this. It's not a big deal though and we can do it in a separate  improvement.

Reply:
I added a TODO, to modify it once we have styles on MainView